students/AntonBakaiev/hw-8/task-3.js

```diff
@@ -0,0 +1,13 @@
+function searchSubstringIndex(initialString, searchingSubstring) {
```

Comment:
These lines of code look pretty the same as [students/tony7kh/hw-8/task-3.js](https://github.com/Temu4/a-level-fe-homeworks-2022/pull/85/files/5b16d96a42797aff0973ad23def89055b38022b4#diff-58f30643081650bb15ec72b949c1261cd4159873004752941a5c1a7f90736756) 🤔

Reply:
@AntonBakaiev @tony7kh please try to create your own solutions in the future)